src/util/createServerRootMixin.js

```diff
@@ -105,7 +105,16 @@ function augmentInstantSearch(instantSearchOptions, cloneComponent) {
           mixins: [
             {
               beforeCreate() {
-                if (component.$nuxt) {
+                const descriptor = Object.getOwnPropertyDescriptor(
+                  this,
```

Comment:
Shouldn't this be `component` ?

```suggestion
                  component,
```